packages/ui/__stories__/Notification.stories.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+
+import { Notification } from '../src/Notification'
+
+const text = 'Lorem ipsum dolor sit amet'
+const longText =
+  'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer ultricies orci sed laoreet tempus. Suspendisse eget semper odio. Nulla vel erat tempor, feugiat nulla vitae, mollis purus. Maecenas varius ante sed mauris scelerisque aliquam. Aenean et congue ante. Etiam dictum, libero sit amet semper posuere, leo metus vestibulum libero, et fringilla metus ante eget lorem. Donec facilisis non nunc at pulvinar. Quisque cursus mi libero, in malesuada nulla tempor et. In at ligula ac est vehicula congue. Donec eget est sed ante cursus malesuada maximus a felis. Phasellus vitae pretium nibh, sed auctor purus. In congue, tortor sed dictum egestas, ligula nibh vehicula nulla, nec euismod ligula diam sit amet mauris. Vivamus congue interdum lorem sit amet facilisis.'
+
+const link = 'Link'
+const linkHref = '/'
+
+export default {
+  title: 'Components/Notification',
+  component: Notification,
+}
+
+export const Success = () => <Notification type="success">{text}</Notification>
+Success.parameters = {
+  design: {
+    type: 'figma',
+    url: 'https://www.figma.com/file/8mVm6LTbp2Z0RaWWjTZoft/%F0%9F%90%9DHIVE-Hazelcast-Design-System?node-id=1735%3A1455',
+  },
+}
+
+export const SuccessWithLongText = () => <Notification type="success">{longText}</Notification>
+
+export const SuccessWithLink = () => (
+  <Notification type="success" link={link} linkHref={linkHref}>
```

Comment:
The chevron is a bit shifted for all links:
![image](https://user-images.githubusercontent.com/17862968/103270268-c05b8c00-49b7-11eb-9e3f-fd4aa7fb990c.png)